code/modules/mob/living/carbon/xenomorph/castes/hunter/abilities_hunter.dm

```diff
@@ -1,6 +1,19 @@
 // ***************************************
 // *********** Stealth
 // ***************************************
+
+//stealth flag defines
+//If hitting an object break stealth.
+#define DIS_OBJ_SLASH (1<<0)
+//If hitting a mob break stealth.
+#define DIS_MOB_SLASH (2<<0)
+//If pouncing at all break stealth.
+#define DIS_POUNCE (3<<0)
+//If pouncing on a target break stealth.
+#define DIS_POUNCE_SLASH (3<<0)
+//If AP is only applied when walking.
+#define WALK_ONLY_AP (4<<0)
```

Comment:
?? What is this? This is not how hunter behaves, this is just a random stealth nerf to hunter

Reply:
![image](https://github.com/user-attachments/assets/1cb6c2bd-0275-4088-a0f0-7a843a94316c)